src/Controller/TpayGetChannelsAction.php

```diff
@@ -33,10 +31,7 @@ public function __invoke(Request $request): Response
         try {
             $tpayResponse = $tpayApi->transactions()->getChannels();
         } catch (TpayException $e) {
-            return new JsonResponse([
-                'error' => $e->getMessage(),
-                'message' => $this->translator->trans('commerce_weavers_sylius_tpay.admin.failed_connection_test', domain: 'flashes', locale: $localeCode),
-            ], Response::HTTP_UNAUTHORIZED);
+            return new JsonResponse(['error' => $e->getMessage()], Response::HTTP_UNAUTHORIZED, ['Accept-Language' => $localeCode]);
```

Comment:
```suggestion
            return new JsonResponse(['error' => $e->getMessage()], Response::HTTP_UNAUTHORIZED, ['Content-Language' => $localeCode]);
```

I think we should rather use Content-Language header as this is a response.